### PR TITLE
Add CUDA 11.8

### PR DESCRIPTION
**Pull request by m3vaz**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by m3vaz** [comment]
@conda-forge-admin, please rerender

**Comment by beckermr** [comment]
@jakirkham Can you check this one and merge it?

**Review by jakirkham — APPROVED** [review]
LGTM

Edit: Bumped the patch version number above

**Comment by jakirkham** [comment]
@jaimergp, could you please review as well?

**Comment by jakirkham** [comment]
Going to go ahead and merge to keep things moving here. Though happy to follow up on anything needed

**Comment by jakirkham** [comment]
Thanks Mark and Matt! 🙏

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% set name = "conda-forge-ci-setup" %}
-{% set version = "3.24.0" %}
+{% set version = "3.24.1" %}
```
Comment (jakirkham) [inline recipe/meta.yaml:2]: Forgot PR ( https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/217 ) used `3.24.1` recently, bumped to `3.24.2` in commit ( https://github.com/conda-forge/conda-forge-ci-setup-feedstock/commit/2b1d2c7835132dd36ca061c538f2904f9a4646d7 )

File: recipe/install_cuda.bat [hunk 2]
```diff
@@ -169,6 +170,14 @@ set "CUDA_INSTALLER_CHECKSUM=a2388d0044b2dd6a3469938eb6108c85"
 set "CUDA_COMPONENTS=%CUDA_COMPONENTS% nsight_nvtx_%CUDA_VERSION%"
 goto cuda_common
 
+:cuda118
+set "CUDA_NETWORK_INSTALLER_URL=https://developer.download.nvidia.com/compute/cuda/11.8.0/network_installers/cuda_11.8.0_windows_network.exe"
+set "CUDA_NETWORK_INSTALLER_CHECKSUM=600ca859835a37395277a5f3a5b6037d"
+set "CUDA_INSTALLER_URL=https://developer.download.nvidia.com/compute/cuda/11.8.0/local_installers/cuda_11.8.0_522.06_windows.exe"
+set "CUDA_INSTALLER_CHECKSUM=acfd3588b31b33aa8c639c718d343c4e"
```
Comment (jakirkham) [inline recipe/install_cuda.bat:177]: In [this CI build]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=620564&view=logs&j=4eca6a2e-7d0b-56b5-2d2a-a8aca6987f82&t=9832422a-f5c0-51e9-a268-92bade59545b ) (also [attached log]( https://github.com/conda-forge/conda-forge-ci-setup-feedstock/files/10135781/nvcc-feedstock_win_64_cuda_compiler_version11.8_20221130.2.txt )), got a checksum mismatch.

When looking at [the checksums here]( https://developer.download.nvidia.com/compute/cuda/11.8.0/docs/sidebar/md5sum.txt ), am also seeing a different value. Think the checksum listed here was for the WSL installer. Given this, think we need to change this one like so:

```suggestion
set "CUDA_INSTALLER_CHECKSUM=894c61ba173d26dc667e95ee734d3c5a"
```

While looking at the checksums, confirmed the network installer checksum was correct.

Comment (m3vaz) [reply recipe/install_cuda.bat:177]: Can do, I'll prep a follow-up patch

Comment (jakirkham) [reply recipe/install_cuda.bat:177]: Submitted PR ( https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/218 )

**Comment by leofang** [comment]
Though it does not explain why CUDA Python 11.7.1 could be compiled fine...

**Comment by jakirkham** [comment]
Maybe some files got shuffled between components?

**Comment by leofang** [comment]
Let's see if #221 would fix it.

**Pull request merged**